README.md

```diff
@@ -121,11 +121,24 @@ The source JavaScript is written in ES6 (located in the `src/js` directory) and
 To version bump the repo and prepare a release, run `npm version major|minor|patch` (following [semver](https://semver.org/)), then (something like) `git push origin HEAD --tags`. It'll automatically update the version number where it appears.
 
 ## Tests
+We would like to get to 100% test coverage and are tracking our progress in this project: [BookReader Fidelity](https://github.com/internetarchive/bookreader/projects/5)
 
-Tests use [QUnit](https://qunitjs.com/). To run all tests, start up an HTTP server at the root of the project and navigate to `/tests/qunit.html` in a browser.
+### End to end tests
+We also have end to end tests using [Testcafe](https://devexpress.github.io/testcafe/documentation/getting-started/).  We write tests for the repo itself and also for our use on archive.org. You can read about them in [here](./tests/e2e/README.md). These are relatively easy to do, and a fantastic way of getting introduced to the wonders of BookReader.  Check the project board for open tickets to work on.  And if you don't see a test for something you spotted, feel free to make an issue.
+
+### Unit tests
+We have unit tests and use Jest to run them.
+For mocks, we use Jest's internal mocking mechanism and Sinon to set spies.
+
+### QUnit tests
+We also have tests that use [QUnit](https://qunitjs.com/). To run all tests, start up an HTTP server at the root of the project and navigate to `/tests/qunit.html` in a browser.
```

Comment:
We can add a comment about being able to run `npm run serve` instead of leaving it up to the individual to figure out a technology to serve the static directory.

Reply:
`npm run serve-dev` is the one to use (should probably rename this to just `serve` though).

Reply:
done